engine/execution/state/delta/delta.go

```diff
@@ -51,19 +51,9 @@ func (d Delta) Set(owner, controller, key string, value flow.RegisterValue) {
 	}
 }
 
-// Delete records a deletion in this delta.
-func (d Delta) Delete(owner, controller, key string) {
-	k := toString(owner, controller, key)
-	d.Data[k] = flow.RegisterEntry{
-		Key:   toRegisterID(owner, controller, key),
-		Value: nil,
-	}
-}
-
```

Comment:
Are we sure about removing this? I thought we kept this as part of the API for the future supporting of deletion

Reply:
The delete on the delta was not used anywhere except in the delete of the view. I figured these were two code paths that are doing the same thing, so any changes in one would also need to be made on the other. That is why I thought removing the explicit delete would simplify the code.

I didn't know about any future plan for supporting deletion. I can re-add this.

Reply:
if we are not using it all good on my side